src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -218,25 +303,42 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
     const newImage = this.hass.states[this._config.entity]?.attributes
       .entity_picture;
 
-    if (!newImage || newImage === oldImage) {
+    if (!newImage) {
       this._image = newImage;
+      this.foregroundColor = undefined;
+      this.backgroundColor = undefined;
       return;
     }
 
-    if (newImage.substr(0, 1) !== "/") {
```

Comment:
So this was causing issues with CORS when the image is from HTTPS and the server is running on HTTP